webbpsf/webbpsf_core.py

```diff
@@ -1564,6 +1720,10 @@ def show_notebook_interface(instrument):
     from IPython.display import display, clear_output
     from matplotlib import pyplot as plt
 
+
+    if isinstance(instrument, string_types):
```

Comment:
Why? For the notebook interface it seems much preferable to just say 
```
webbpsf.show_notebook_interface("NIRCam")
```
rather than
```
nc = webbpsf.NIRCam()
webbpsf.show_notebook_interface(nc)
```
The latter is more verbose but provides zero additional functionality.